pages/docs/dev-docs/dev-development.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Development setup
+keywords: pages, development
+permalink: dev-docs-dev-development.html
+---
+
+## Getting started
```

Comment:
Very helpful hints for students. I wanted since long as well to add such a guide somewhere.